.github/workflows/deploy-to-cloud-run.yml

```
@@ -0,0 +1,172 @@
+          echo "CLOUDSQL_INSTANCE_NAME=${{ secrets.CLOUDSQL_INSTANCE_NAME }}" >> .env
+
+      - name: Authenticate to Google Cloud
+        uses: google-github-actions/auth@v2
+        with:
+          credentials_json: ${{ secrets.GCP_SA_KEY }}
+
+      - name: Set up Google Cloud SDK
+        uses: google-github-actions/setup-gcloud@v2
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          export_default_credentials: true
```

actionlint (1.7.7)

input "export_default_credentials" is not defined in action "google-github-actions/setup-gcloud@v2". available inputs are "install_components", "project_id", "skip_install", "version"

(action)